docs/users-manual/env-of-job.rst

```diff
@@ -253,6 +253,19 @@ Singularity sandbox image (an exploded directory).  :tool:`condor_submit`
 will parse this image and advertise what type of container image it
 is, and match with startds that can support that image.
 
+Note that the container image, like most other submit file commands,
+contain a $$ expansion.  This may be useful if you would like to use 
```

Comment:
```suggestion
Note that :subcom:`container_image`, like most other submit file commands,
can contain a ``$$`` expansion.  This may be useful if you would like to use 
```